From: Randy Dunlap <rdunlap@infradead.org>
Date: Fri, 22 Sep 2017 18:04:43 -0700
Subject: nvme: use menu Kconfig interface
Patch-mainline: v4.15-rc1
Git-commit: 786456325b9f05843ec722269a3ccd77c31b9c6d
References: FATE#323952, FATE#322506

Add a menu interface for NVME host and target support so that it is
presented to users more like other Kconfig symbols.
This makes the Device Driver menu less cluttered (easier to read)
and keeps all of these symbols grouped together.

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: Keith Busch <keith.busch@intel.com>
Reviewed-by: Sagi Grimberg <sagi@grimberg.me>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 drivers/nvme/Kconfig |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/nvme/Kconfig
+++ b/drivers/nvme/Kconfig
@@ -1,2 +1,6 @@
+menu "NVME Support"
+
 source "drivers/nvme/host/Kconfig"
 source "drivers/nvme/target/Kconfig"
+
+endmenu
